documentation/configuration/kelb.md

```diff
@@ -0,0 +1,6 @@
+# KELB Configuration
+
+## Environment variables
+
+- `OTEL_METRICS_EXPORT_DESTINATION`: Http endpoint to which the otlp metrics should be delivered. No value sends metrics to stdout instead.
```

Comment:
I would have definitely named these `OTLP_*`, this name confused me at first. But not a blocker for me.